src/Stratis.Bitcoin.Features.MemoryPool/MempoolBehavior.cs

```diff
@@ -366,6 +371,21 @@ public override object Clone()
             this.logger.LogTrace("(-)");
         }
 
+        /// <summary>
+        /// Whether peer should receive only blocks.
+        /// </summary>
+        /// <param name="peer">Peer to check.</param>
+        /// <returns>Whether peer should receive only blocks.</returns>
+        private bool IsBlocksOnly(INetworkPeer peer)
+        {
+            this.logger.LogTrace("({0}:'{1}'", nameof(peer), peer.RemoteSocketEndpoint);
+
+            bool isBlocksOnly = !this.connectionManager.ConnectionSettings.RelayTxes && !this.isPeerWhitelistedForRelay;                
```

Comment:
This is not suppose to change so why not do it as a parameter as well similar to `isPeerWhitelistedForRelay`